packer/ansible/roles/aggregate/files/motd/10-services

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Given that Tomcat and PostgreSQL are not configurable, does the user really need to know that both are running on so and so port?

Reply:
I'll remove it on my next changeset with gradle tasks :) It's almost ready